templates/zerver/emails/new_login/new_login_alert.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
+<html lang="en">
+<head>
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+        <title>Zulip</title>
+</head>
+<body>
+    <table width="80%" style="align:center; max-width:800px" align="center">
+        <tr>
+            <td>
+                <a href="{{ realm_uri }}/"><img style="max-height:75px; height:75px;"  height="75px" alt="Zulip" title="{{ _('Zulip') }}" src="{{ realm_uri }}/static/images/landing-page/zulip-header.png" /></a>
+                <h3 style="font-family:Arial; font-size:30px; margin: 5px 0px; color:#555">New Login To Zulip</h3>
```

Comment:
Generally we only capitalize proper nouns and first letters in things, so maybe "login" rather than "Login"?